docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2.1'
+services:
+  postgres:
+    image: postgres:9.3
+
+  possum:
+    image: registry2.itci.conjur.net/possum:0.1.0-stable
```

Comment:
Let's just call this `registry.tld` everywhere. To set this up all you have to do is 1. add this to your `/etc/hosts` file

   ```
   34.203.67.165 registry registry.tld 
   ```

2. Run your Docker daemon with `registry.tld` as an insecure registry. Already should be, from the v1 registry.
3. `docker stop conjur-registry-proxy` (yay!)